### PR TITLE
Refactor Dashboard.js States and Props; Filter Employee Logic

### PR DESCRIPTION
**Pull request by Kyle7286**
- Refactored how Dashboard.js States are initialized; 
- Refactored how props are used to send the state data from dashboard to the sub-components

Summary of changes:
- Adjusted the Dashboard.js states for users, clients & tickets to include the proper nested object/arrays
- Changed the props to send the states we adjusted, removing the ternary logic in the props.
- Filter employee logic
divvy2

In summary:
- The ternaries in the props was causing issues with data compiling.
- Need to set the states up properly first then just send the state down and all the components are happy.

image
image
image
image

### CODE REVIEW

**Review by wryanj — APPROVED** [review]
Apoproved- will merge with my local and review to make passing of comments data consistent. May add a little extra time to comments just bc I have to review and maybe re-do a few short things.
